### PR TITLE
Update community show layout

### LINKED ISSUE #98: Update the communities show actions
We need to update the community show page to  handle different viewing scenarios.

If there is are no finalized indexes for the community we should show a message about starting a new one.

If there is an index show render the dashboard for the latest finalized index.

### PR DESCRIPTION
**Pull request by teknofire**
First stab at getting the community show page all nice looking

This closes #98 

Not yet ready for review

### CODE REVIEW

**Comment by teknofire** [comment]
image

Screenshot of the initial layout, this community example shows what would happen if there multiple dashboards for a community, one that is published and one that is still in progress

I'm thinking that I want to move the sidebar to the left side

**Comment by teknofire** [comment]
Same community but for a guest user

image

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,9 @@ gem 'cancancan'
 gem 'rgeo'
 gem 'rgeo-geojson'
 gem 'rails_12factor', group: :production
+gem 'high_voltage', github: 'thoughtbot/high_voltage'
+gem 'georuby', require: ['georuby', 'geo_ruby/ewk']
```
Comment (smacfarlane) [inline Gemfile:31]: Ack!  georuby and rgeo?  Do we actually need both?

Comment (teknofire) [inline Gemfile:31]: oh weird, I looked through the gem file to see if one or the other was in there and didn't see anything.

Comment (teknofire) [inline Gemfile:31]: Right now, I'm leaning towards wanting to remove rgeo as it's provides a lot of extra functionality that we don't use like trying to reproject coordinates.  Where as georuby is much lighter weight and just handles getting reading and writing geospatial data.

I think that would be better done as a separate issue to, as I'll need to update the db:seed stuff

Comment (grimm) [inline Gemfile:31]: Probably my fault as I added the rgeo stuff to import the communities.  :(

Comment (teknofire) [inline Gemfile:31]: No persons fault, things evolve!

**Comment by teknofire** [comment]
Update the layout, I think this is a bit cleaner, let me know what people think

image

**Comment by teknofire** [comment]
the mini-dashboards have also been updated with bars for now
image

**Comment by grimm** [comment]
:+1:  Looks good.

**Comment by teknofire** [comment]
@gina-alaska/awrvi ready for review

**Comment by smacfarlane** [comment]
Not keen on the word dashboard, but I think that's something we need to push up to the AWRVI researchers. 

:+1:

**Comment by vlraymond** [comment]
mini-dashboards:
The wording for the "AWRVI 0.42" part should be more obvious somehow. 
AWRVI Vulnerability Score: 0.42

Dream item:
"According to this data, your community is ________ (vulnerable / moderately vulnerable)"

Still don't like the colors.

**Pull request merged**